SMTX/SMTX.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
```

Comment:
I think you don't need this project anymore?

Reply:
oh boi, didn't see I left this, accidente